cli/testdata/spinner_cases/stop.ts

```diff
@@ -4,4 +4,5 @@ const spinner = new Spinner();
 
 spinner.start();
 
-setTimeout(spinner.stop, 200);
+setTimeout(() => spinner.stop(), 200);
+setTimeout(() => {}, 500);
```

Comment:
Wait another 300ms (500ms - 200ms) to see if the spinner doesn't output text anymore